app/decorators/container_manager_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class ManageIQ::Providers::ContainerManager < MiqDecorator
```

Comment:
This file needs to be in `app/decorators/manageiq/providers/container_manager_decorator.rb` (simple rule, the path to the decorator must fit the path to the model).